machine/v1/types_nutanixprovider.go

```diff
@@ -56,6 +56,22 @@ type NutanixMachineProviderConfig struct {
 	// +kubebuilder:validation:Required
 	SystemDiskSize resource.Quantity `json:"systemDiskSize"`
 
+	// bootType indicates the boot type (SECURE_BOOT, UEFI or LEGACY) the VM should use to boot.
+	// If this field is empty, the default boot type LEGACY is used for the VM to boot.
+	// SECURE_BOOT depends on UEFI boot, i.e., enabling SECURE_BOOT means that UEFI boot is also enabled.
+	// +kubebuilder:validation:Enum="";LEGACY;UEFI;SECURE_BOOT
+	// +optional
+	BootType NutanixBootType `json:"bootType,omitempty"`
+
+	// Project optionally identifies a Prism Central project for the Machine's VM to associate with.
```

Comment:
Can you extend this comment to explain what happens when this is omitted please

Reply:
Note that L60 explains:

// If this field is empty, the default boot type LEGACY is used for the VM to boot.

Reply:
done